src/Effectful/Error.hs

```diff
@@ -1,4 +1,20 @@
--- | Support for checked exceptions.
+{- | Support for checked exceptions.
+
+   The `Error` effect provided by this module is a suitable replacement for the
+   `Control.Monad.Except.ExceptT` monad transformer found in the mtl library.
+   It is __not__ intended to be a general mechanism for catching errors like
+   the functions of the 'exceptions' package. For example, if you want to
+   a `Control.Exception.SomeException` you could rely on `Control.Monad.catch`:
```

Comment:
I think you accidentally a word ;)

Reply:
I'm afraid I don't quite understand your comment ;-)

Reply:
You wrote "if you want to a `Control.Exception.SomeException`", there's a missing "catch" there :slightly_smiling_face:

Reply:
:D Well, forgot to catch that exception... Fixed.